Containerfile.cert-manager-operator.bundle

```diff
@@ -10,9 +10,9 @@ COPY --chmod=0550 hack/bundle/render_templates.sh /render_templates.sh
 # For image builds through konflux, konflux-bot will update the references.
 
 ARG CERT_MANAGER_OPERATOR_IMAGE=registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:bbf2fbcaa5751108a6babc0a74c509d35324994f7414f9293b9d8e6773404007 \
-    CERT_MANAGER_WEBHOOK_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:b0a2605fd95ec03a00cd4003590eecf9d27ba3285991a84a59e738821da45ba8 \
-    CERT_MANAGER_CA_INJECTOR_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:b0a2605fd95ec03a00cd4003590eecf9d27ba3285991a84a59e738821da45ba8 \
-    CERT_MANAGER_CONTROLLER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:b0a2605fd95ec03a00cd4003590eecf9d27ba3285991a84a59e738821da45ba8 \
+    CERT_MANAGER_WEBHOOK_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:9da3aa20b0313fe6cffd088bc0f7d2da13ba7f4cae1bc5f9483bae47ba2770ed \
```

Comment:
Same query posted in #78 an #79 applies in this change set.

Reply:
same as https://github.com/openshift/cert-manager-operator-release/pull/79#discussion_r1879574775